External/FEXCore/include/FEXCore/IR/RegisterAllocationData.h

```diff
@@ -28,7 +28,8 @@ union PhysicalRegister {
 
 static_assert(sizeof(PhysicalRegister) == 1);
 
-class RegisterAllocationData {
+// This class is serialized, can't have any holes in the structure otherwise ASAN complains
```

Comment:
```suggestion
// This class is serialized, can't have any holes in the structure otherwise ASAN complains about reading uninitialized memory
```

I can guess, but better to be explicit how they're related :)

Reply:
Resolved.